aidants_connect_carto/apps/web/templates/layouts/nav.html

```diff
@@ -13,6 +13,11 @@ <h2>&nbsp;Carto</h2>
             Tous les lieux
           </a>
         </li>
+        <li class="nav__item">
+          <a href="{% url 'data-sources-list' %}"  {% if request.path == '/datasets' %}class="active"{% endif %}>
+            Donnée
```

Comment:
Ça me parait bizarre comme intitulé.
```suggestion
            Jeux de données
```